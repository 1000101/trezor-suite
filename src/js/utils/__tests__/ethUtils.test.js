describe('sds', () => {
    it('decimalToHex', () => {
        expect(2).toMatchSnapshot();
    });
});
