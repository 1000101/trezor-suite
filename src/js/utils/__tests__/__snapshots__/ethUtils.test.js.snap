// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sds decimalToHex 1`] = `2`;
